database/server.go

```diff
@@ -489,8 +492,8 @@ func UpsertServiceName(serverName string, typ server_structs.ServerType) error {
 }
 
 // Retrieve the server name in use - lookup the entry whose UpdatedAt is the most recent
-func GetServiceName() (string, error) {
-	var entry server_structs.ServiceName
+func GetServerName() (string, error) {
```

Comment:
What mechanism makes the most recently updated serverName the one for this server? If this function is used on the registry would it give the intended results?

Noting this isn't used anywhere might delete it to save time.